ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/IOptimizeService.java

```diff
@@ -117,4 +118,8 @@ public interface IOptimizeService {
    * @param toRemoveTables -
    */
   void clearRemovedTables(List<TableIdentifier> toRemoveTables);
+
+  void enableSelfOptimizing(ArcticTable table);
+
+  void disableSelfOptimizing(ArcticTable table);
```

Comment:
We'd better remove `enableSelfOptimizing` and `disableSelfOptimizing` from the interface `IOptimizeService`, and make them internal logic of `OptimizeService`.

And I think both `enableSelfOptimizing` and `disableSelfOptimizing` can be implemented in `refreshAndListTables`.